docs/language/operators.md

```diff
@@ -1094,3 +1095,46 @@ zq -z 'cut District,County | uniq' schools.zson
 {District:"Ojai Unified",County:"Ventura"}
 ...
 ```
+
+---
+
+## `yield`
+
+|                           |                                                 |
+| ------------------------- | ----------------------------------------------- |
+| **Description**           | For each input value, produces one or more values downstream.
+| **Syntax**                | `yield <expression> [, (<expression>)...]` |
+| **Required arguments**    | One or more of `<expression>`, which may be any valid Zed [expression](expressions.md).
+| **Optional arguments**    | None |
```

Comment:
I know this is following the convention established in this file, but I'd prefer following the way (over is documented)[https://github.com/brimdata/zed/blob/main/docs/language/operators.md#over]. The tables look nice when the markdown is rendered but the markdown itself can become unwieldy to edit. See the table in the [join section](https://github.com/brimdata/zed/blob/main/docs/language/operators.md#join).

Reply:
I am planning at some point to reformat these to get rid of the table.

Reply:
I was following convention and didn't realize there is precedent for not following it.  Let's do the change in the subsequent PR as suggested above.